Exercises/Modules/Chapter 18/Soln18_01/Box.cppm

```diff
@@ -0,0 +1,27 @@
+export module box;
+
+import <ostream>;
+import <format>;
+
+export class Box
+{
+public:
+  Box() = default;
+  Box(double length, double width, double height) 
+    : m_length{length}, m_width{width}, m_height{height} {};
+
+  double volume() const 
+  {
+    return m_length * m_width * m_height;
+  }
+
+  friend std::ostream& operator<<(std::ostream& out, const Box& box)
```

Comment:
Why friend? You usually never implement op<< as friend.

Reply:
Usually yes, but this Box does not have getters. You've got to change things up from time to time, otherwise it get's boring ;-)